storage/addressing.go

```diff
@@ -62,6 +62,13 @@ func updateRangeAddressing(b *client.Batch, desc *roachpb.RangeDescriptor) error
 	return rangeAddressing(b, desc, putMeta)
 }
 
+func makeMetaKey(prefix roachpb.Key, key roachpb.RKey) roachpb.Key {
```

Comment:
shouldn't this live in `keys`? I think it's worthwhile not to create ad-hoc keys anywhere in the codebase.